x/subspaces/types/models.go

```diff
@@ -84,9 +84,22 @@ func (sub Subspace) Validate() error {
 		return fmt.Errorf("invalid subspace creation time: %s", sub.CreationTime)
 	}
 
+	err = sub.AllowedFeeTokens.Validate()
+	if err != nil {
+		return fmt.Errorf("invalid allowed fee tokens: %s", sub.AllowedFeeTokens)
+	}
+
 	return nil
 }
 
+// Set allowed fee tokens without validating it.
+// Before storing the updated subspace, a validation with Validate() should
+// be performed.
+func (sub Subspace) SetAllowedFeeTokens(feeTokens sdk.Coins) Subspace {
```

Comment:
I'm not a big fan of this approach. Instead I think it would be better to: 

1. Add another param inside the `NewSubspace` function to specify the fee tokens when creating the subspace. This is going to be useful when exporting/importing the genesis as well as inside tests. 
2. Rename this method `Update` as we have done with the `Post` and `Attachment` type, to keep everything consistent.

Also, I'm doubting whether `AllowedFeeTokens` is the proper name. I think `AdditionalFeeTokens` might be better instead. The name `AllowedFeeTokens` might make people think that if I specify `0.01ustar` then the `star` token is going to be then only one allowed, while the `udsm` should always be allowed anyway. Using `AdditionalFeeTokens` might make everything more easy to understand.

Reply:
I see, I use the `SetAllowedFeeTokens` since adding another param inside the `NewSubspace` will break hundreds of existing test codes, but yeah, it is better to have an additional field.

strongly agree with `AdditionalFeeTokens` is better.

Reply:
Finally, using `sed` to update the new field quickly:
```bash
#!/bin/bash

dir="./x"

find "$dir" -type f -name "*.go" | while read -r file; do
    sed -i -E ':a;N;$!ba;s/NewSubspace\([^)]*blockTime,/&\nnil,/g' "$file"
    echo "$file"
done
```

Reply:
I honestly don't know of any other tool that can easily update the code. I would have done it by hand :sweat: Great job using `sed` :100: